test/bugzilla.d

```diff
@@ -0,0 +1,30 @@
+import utils;
+
+@("after-merge-close-issue-bugzilla")
+unittest
+{
+    setAPIExpectations(
+        "/github/repos/dlang/phobos/pulls/4963/commits", (ref Json j) {
+            j[0]["commit"]["message"] = "Fix Issue 17564";
+         },
+        "/github/repos/dlang/phobos/issues/4963/comments",
+        "/github/repos/dlang/phobos/issues/4963/labels",
+        (scope HTTPServerRequest req, scope HTTPServerResponse res){
+            res.writeJsonBody(Json.emptyArray);
+        },
+        "/github/repos/dlang/phobos/issues/4963/labels",
+        (scope HTTPServerRequest req, scope HTTPServerResponse res){
+            res.writeVoidBody;
+        },
```

Comment:
We should probably don't rewrite the comment and update the labels when a PR gets merged (that's why I had to add these three expectations), but this can be done in another PR.